model/registry/incident.yaml

```diff
@@ -0,0 +1,25 @@
+groups:
+  - id: registry.incident
+    prefix: incident
+    type: attribute_group
+    brief: >
+     This group describes attributes specific to incidents.
+    attributes:
+      - id: name
+        type: string
+        stability: experimental
+        brief: >
+          The name of the incident.
+        examples: ['incident-123']
+      - id: id
+        type: string
+        stability: experimental
+        brief: >
+          The id of the incident.
+        examples: ['123']
```

Comment:
I think you're right that this is better served by a URL instead of ID. Incident has been moved out to be its own iterative issue #1185 that will be worked on separately from this PR. Will address those aspects through that issue.